openfe/protocols/openmm_utils/omm_settings.py

```diff
@@ -371,3 +370,58 @@ def must_be_positive(cls, v):
                       "intervals must be positive")
             raise ValueError(errmsg)
         return v
+
+
+class RepeatSettings(SettingsBaseModel):
+    """Settings for how many independent MD runs to perform."""
+
+    n_repeats: int = 1
+    """
+    Number of independent repeats to run.  Default 1
+    """
+
+
+class SimulationSettingsMD(SimulationSettings):
+    """
+    Settings for simulation control for plain MD simulations, including
+    writing outputs
+    """
+    class Config:
+        arbitrary_types_allowed = True
+
+    equilibration_length_nvt: unit.Quantity
+    """
+    Length of the equilibration phase in the NVT ensemble in units of time. 
+    The total number of steps from this equilibration length
+    (i.e. ``equilibration_length_nvt`` / :class:`IntegratorSettings.timestep`)
+    must be a multiple of the value defined for
+    :class:`IntegratorSettings.n_steps`.
+    """
+    # reporter settings
+    output_filename = 'simulation.xtc'
+    """Path to the storage file for analysis. Default 'simulation.xtc'."""
+    trajectory_interval = 5000 * unit.timestep
+    """
+    Frequency to write the xtc file. Default 5000 * unit.timestep.
+    """
+    output_structure = 'system.pdb'
+    """Path to the pdb file of the full pre-minimized system. Default 'system.pdb'."""
+    minimized_structure = 'minimized.pdb'
+    """Path to the pdb file of the system after minimization. 
+    Only the specified atom subset is saved. Default 'minimized.pdb'."""
+    equ_NVT_structure = 'equ_NVT.pdb'
```

Comment:
```suggestion
    equil_NVT_structure = 'equ_NVT.pdb'
```
Extra two characters is probably worth it for non-native speakers